recipes/Hapi/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "Hapi" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/Hapi/archive/{{ version }}.tar.gz
+  sha256: 112e4e14562233b75f1157ceb8bb01c0ac33b4f469c28af8fcc1c2bf080e6775
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+
+  host:
+    - python >=2.7
+    - pip  
+    - numpy
+    - affine
+    - fiona
+    - proj
+    - pyproj
+    - gdal
+    - pandas
+    - geopandas
+    - matplotlib
+    - python
+    - scipy
+    - shapely
+    - statsmodels
+    - rasterio
+    - rasterstats
+    - netCDF4
+    - scikit-learn
+    - scikit-image
+    - ecmwf-api-client
+    - oasis
+    - joblib
```

Comment:
You probably only need python and pip here. Unless the package imports itself in the setup.py. If that is the case please open an issue upstream b/c that usually leads to bad things happening.

Reply:
@conda-forge/help-python  @conda-forge/help-python-c so what should I do here ?, if I remove those packages the recipe will not pass the test they are needed in the package

Reply:
`run` are the package requirementes, host are build requirements, I'm sure those are not needed to build it, unless the package imports itself in setup.py. Let's test it...

Reply:
@ocefpaf it passed the checks